lesson_6/homework_6_1.py

```diff
@@ -0,0 +1,11 @@
+my_input = input("Please enter the text: ")
+
+all_symbols_in_my_input = set(my_input)
+
+unique_symbols_counter = 0
+
+for symbol in all_symbols_in_my_input:
+    if my_input.count(symbol) == 1:
+        unique_symbols_counter += 1
```

Comment:
set already has unique symbols, so every symbol counter == 1

Reply:
on line 3 you convert input string to set, it contains only unique letters.
`if my_input.count(symbol) == 1` so this will always be true

just use len()

Reply:
нее.. задум тут у іншому, сет у такому випадку містить лише ті символи, які використовуються у моєму інпуті.
і я циклом проходжу по цім символам і рахую їх кілкість у інпуті, якщо кількість більше 1 то цей символ не є унікальним у мойому інпуті, а якщо він у інпуті зустрічається тільки один раз - то він є унікальним в інпуті.
наприклад інпут = "qqwweerrttyyuuiiooppaa" не вмістить ні одного унікального символа = всі повторюются по два рази - і тоді len(all_symbolls_in_my_input) дасть 11 - але це не є True по задачі по унікальності.

Reply:
> наприклад інпут = "qqwweerrttyyuuiiooppaa" не вмістить ні одного унікального символа = всі повторюются по два рази

отут конфуз
в цьому рядку унікальними вважаються qwertyuiopa
не малося на увазі що треба рахувати той який 1 раз зустрічається